include/mbgl/storage/default_file_source.hpp

```diff
@@ -195,6 +190,11 @@ class DefaultFileSource : public FileSource {
      */
     void packDatabase(std::function<void(std::exception_ptr)> callback);
 
+    /*
+     * Enables or disables auto database file packing after deleteing regions and clearing ambient cache.
+     */
+    void setAutopack(bool);
+
     /*
      * Forces revalidation of the ambient cache.
      *
```

Comment:
If I'm understanding this PR correctly, we are refactoring `clearAmbientCache()` so that VACUUM is not always automatically run.

Therefore the documentation of `clearAmbientCache()` on lines 215-217 should be updated.